### ISSUE TITLE
[Bug]: OCP3.3.0 failed to deploy because of table meta_database.compute_vpc doesn't exist

### ISSUE THREAD

**Issue by raygift** [oceanbase/oceanbase#980]
### Please check the issue list for the same bug

- [X] I have checked issue list and this bug is encountered for the first time.

### Environment

```markdown
- OCP 3.3.0
- CentOS 7.9
```

### Fast Reproduce Steps(Required)

When install ocp following [ocp docs](https://open.oceanbase.com/docs/ocp-cn/V3.3.0/10000000000202851)，after modified the config.yaml，got error when execute installation with command `./ocp_installer.sh install -c config.yaml -k /root/.ssh/id_rsa -i ./ocp-installer.tar.gz -o ./ocp.tar.gz` 

### Actual Behavior

```
all sql files:
['ocp_metadb_ddl.sql', 'ocp_metadb_tablegroup.sql', 'ocp_metadb_dml.sql', 'update_ddl_2.2.0.sql', 'update_ddl_2.3.0.sql', 'update_ddl_2.3.1.sql', 'update_ddl_2.3.2.sql', 'update_ddl_2.3.3.sql', 'update_ddl_2.3.4.sql', 'update_ddl_2.3.5.sql', 'update_ddl_2.3.7.sql', 'update_ddl_2.4.0.sql', 'update_ddl_2.4.1.sql', 'update_ddl_2.4.2.sql', 'update_ddl_2.4.3.sql', 'update_ddl_2.4.4.sql', 'update_ddl_2.4.5.sql', 'update_ddl_2.5.0.sql', 'update_ddl_2.5.1.sql', 'update_ddl_2.5.4.sql', 'update_ddl_3.1.0.sql', 'update_ddl_3.1.1.sql', 'update_ddl_3.1.2.sql', 'update_ddl_3.2.0.sql', 'update_dml_2.2.0.sql', 'update_dml_2.3.0.sql', 'update_dml_2.3.1.sql', 'update_dml_2.3.2.sql', 'update_dml_2.3.3.sql', 'update_dml_2.3.4.sql', 'update_dml_2.3.5.sql', 'update_dml_2.3.6.sql', 'update_dml_2.3.7.sql', 'update_dml_2.4.0.sql', 'update_dml_2.4.1.sql', 'update_dml_2.4.2.sql', 'update_dml_2.4.3.sql', 'update_dml_2.4.4.sql', 'update_dml_2.4.5.sql', 'update_dml_2.5.0.sql', 'update_dml_2.5.1.sql', 'update_dml_2.5.2.sql', 'update_dml_2.5.3.sql', 'update_dml_2.5.4.sql', 'update_dml_3.1.0.sql', 'update_dml_3.1.1.sql', 'update_dml_3.1.2.sql', 'update_dml_3.1.3.sql', 'update_dml_3.2.0.sql', 'update_ddl_3.2.1.sql', 'update_dml_3.2.1.sql', 'update_ddl_3.2.2.sql', 'update_ddl_3.2.3.sql', 'update_ddl_3.3.0.sql', 'update_dml_3.2.2.sql', 'update_dml_3.2.3.sql', 'update_dml_3.3.0.sql', 'generate/audit_event_meta.sql', 'generate/config_properties.sql', 'generate/config_properties_deleted.sql', 'generate/iam_role.sql', 'generate/inspection_script.sql', 'generate/mc_ocp_cluster_config.sql', 'generate/metric_class.sql', 'generate/metric_class_deleted.sql', 'generate/metric_expression.sql', 'generate/metric_expression_deleted.sql', 'generate/metric_group.sql', 'generate/metric_group_deleted.sql', 'generate/metric_meta.sql', 'generate/metric_meta_deleted.sql', 'generate/monitor_metric_ob_collect_config.sql', 'generate/ob_cluster_event_info.sql', 'generate/ob_error_code.sql', 'generate/ob_parameter_info.sql', 'generate/ob_system_variable_info.sql', 'generate/obproxy_parameter_info.sql', 'generate/ocp2_alarm_metric_meta.sql', 'generate/ocp2_alarm_metric_meta_deleted.sql', 'generate/ocp2_alarm_rule.sql', 'generate/ocp2_alarm_rule_deleted.sql', 'generate/ocp2_template_variable_meta.sql', 'generate/ocp_alarm_detect_rule.sql', 'generate/ocp_alarm_detect_rule_deleted.sql', 'generate/ocp_alarm_group.sql', 'generate/ocp_alarm_group_alarm_type.sql', 'dynamic_config_properties.sql']
start to load sqls
replace table_group in sql file: ocp_metadb_ddl.sql
sed 's/$VAR_TABLEGROUP_NAME/meta_database/g' ../../sqls/ocp_metadb_ddl.sql > ../../sqls/real/ocp_metadb_ddl.sql
executing real sql script: real/ocp_metadb_ddl.sql
replace table_group in sql file: ocp_metadb_tablegroup.sql
sed 's/$VAR_TABLEGROUP_NAME/meta_database/g' ../../sqls/ocp_metadb_tablegroup.sql > ../../sqls/real/ocp_metadb_tablegroup.sql
executing real sql script: real/ocp_metadb_tablegroup.sql
replace table_group in sql file: ocp_metadb_dml.sql
sed 's/$VAR_TABLEGROUP_NAME/meta_database/g' ../../sqls/ocp_metadb_dml.sql > ../../sqls/real/ocp_metadb_dml.sql
executing real sql script: real/ocp_metadb_dml.sql
replace table_group in sql file: update_ddl_2.2.0.sql
sed 's/$VAR_TABLEGROUP_NAME/meta_database/g' ../../sqls/update_ddl_2.2.0.sql > ../../sqls/real/update_ddl_2.2.0.sql
executing real sql script: real/update_ddl_2.2.0.sql
replace table_group in sql file: update_ddl_2.3.0.sql
sed 's/$VAR_TABLEGROUP_NAME/meta_database/g' ../../sqls/update_ddl_2.3.0.sql > ../../sqls/real/update_ddl_2.3.0.sql
executing real sql script: real/update_ddl_2.3.0.sql
run sql alter table ocp_rpm_svn add column upgrade_post_checker varbinary(262144); got duplicate column error 1060 (42S21): Duplicate column name 'upgrade_post_checker', just skip
replace table_group in sql file: update_ddl_2.3.1.sql
sed 's/$VAR_TABLEGROUP_NAME/meta_database/g' ../../sqls/update_ddl_2.3.1.sql > ../../sqls/real/update_ddl_2.3.1.sql
executing real sql script: real/update_ddl_2.3.1.sql
replace table_group in sql file: update_ddl_2.3.2.sql
sed 's/$VAR_TABLEGROUP_NAME/meta_database/g' ../../sqls/update_ddl_2.3.2.sql > ../../sqls/real/update_ddl_2.3.2.sql
executing real sql script: real/update_ddl_2.3.2.sql
replace table_group in sql file: update_ddl_2.3.3.sql
sed 's/$VAR_TABLEGROUP_NAME/meta_database/g' ../../sqls/update_ddl_2.3.3.sql > ../../sqls/real/update_ddl_2.3.3.sql
executing real sql script: real/update_ddl_2.3.3.sql
replace table_group in sql file: update_ddl_2.3.4.sql
sed 's/$VAR_TABLEGROUP_NAME/meta_database/g' ../../sqls/update_ddl_2.3.4.sql > ../../sqls/real/update_ddl_2.3.4.sql
executing real sql script: real/update_ddl_2.3.4.sql
replace table_group in sql file: update_ddl_2.3.5.sql
sed 's/$VAR_TABLEGROUP_NAME/meta_database/g' ../../sqls/update_ddl_2.3.5.sql > ../../sqls/real/update_ddl_2.3.5.sql
executing real sql script: real/update_ddl_2.3.5.sql
replace table_group in sql file: update_ddl_2.3.7.sql
sed 's/$VAR_TABLEGROUP_NAME/meta_database/g' ../../sqls/update_ddl_2.3.7.sql > ../../sqls/real/update_ddl_2.3.7.sql
executing real sql script: real/update_ddl_2.3.7.sql
replace table_group in sql file: update_ddl_2.4.0.sql
sed 's/$VAR_TABLEGROUP_NAME/meta_database/g' ../../sqls/update_ddl_2.4.0.sql > ../../sqls/real/update_ddl_2.4.0.sql
executing real sql script: real/update_ddl_2.4.0.sql
replace table_group in sql file: update_ddl_2.4.1.sql
sed 's/$VAR_TABLEGROUP_NAME/meta_database/g' ../../sqls/update_ddl_2.4.1.sql > ../../sqls/real/update_ddl_2.4.1.sql
executing real sql script: real/update_ddl_2.4.1.sql
run sql ALTER TABLE `ocp2_template_variable_meta` ADD `description3` varchar(512) NULL COMMENT 'zh_TW 变量描述'; got duplicate column error 1060 (42S21): Duplicate column name 'description3', just skip
run sql ALTER TABLE `ocp2_alarm_metric_meta` ADD `name3` varchar(64) NULL COMMENT 'zh_TW 指标名称'; got duplicate column error 1060 (42S21): Duplicate column name 'name3', just skip
run sql ALTER TABLE `ocp2_alarm_metric_meta` ADD `description3` varchar(512) NULL COMMENT 'zh_TW 指标描述'; got duplicate column error 1060 (42S21): Duplicate column name 'description3', just skip
run sql ALTER TABLE `ocp2_alarm_metric_meta` ADD `unit2` varchar(32) NULL COMMENT 'en_US 单位'; got duplicate column error 1060 (42S21): Duplicate column name 'unit2', just skip
run sql ALTER TABLE `ocp2_alarm_metric_meta` ADD `unit3` varchar(32) NULL COMMENT 'zh_TW 单位'; got duplicate column error 1060 (42S21): Duplicate column name 'unit3', just skip
run sql ALTER TABLE `ocp2_alarm_rule` ADD `name3` varchar(64) NULL COMMENT 'zh_TW 告警项名称'; got duplicate column error 1060 (42S21): Duplicate column name 'name3', just skip
run sql ALTER TABLE `ocp2_alarm_rule` ADD `summary_template3` varchar(256) NULL COMMENT 'zh_TW 概述模板'; got duplicate column error 1060 (42S21): Duplicate column name 'summary_template3', just skip
run sql ALTER TABLE `ocp2_alarm_rule` ADD `description_template3` varchar(512) NULL COMMENT 'zh_TW 详情模板'; got duplicate column error 1060 (42S21): Duplicate column name 'description_template3', just skip
run sql ALTER TABLE `ocp2_alarm_channel` ADD `name2` varchar(64) NULL COMMENT 'en_US 名称'; got duplicate column error 1060 (42S21): Duplicate column name 'name2', just skip
run sql ALTER TABLE `ocp2_alarm_channel` ADD `name3` varchar(64) NULL COMMENT 'zh_TW 名称'; got duplicate column error 1060 (42S21): Duplicate column name 'name3', just skip
run sql ALTER TABLE `ocp2_alarm_channel` ADD `message_template3` varchar(4096) NULL COMMENT 'en_US 消息模版'; got duplicate column error 1060 (42S21): Duplicate column name 'message_template3', just skip
run sql ALTER TABLE `ocp2_alarm_channel` ADD `aggregation_message_template3` varchar(4096) NULL COMMENT 'en_US 聚合消息模板'; got duplicate column error 1060 (42S21): Duplicate column name 'aggregation_message_template3', just skip
replace table_group in sql file: update_ddl_2.4.2.sql
sed 's/$VAR_TABLEGROUP_NAME/meta_database/g' ../../sqls/update_ddl_2.4.2.sql > ../../sqls/real/update_ddl_2.4.2.sql
executing real sql script: real/update_ddl_2.4.2.sql
replace table_group in sql file: update_ddl_2.4.3.sql
sed 's/$VAR_TABLEGROUP_NAME/meta_database/g' ../../sqls/update_ddl_2.4.3.sql > ../../sqls/real/update_ddl_2.4.3.sql
executing real sql script: real/update_ddl_2.4.3.sql
replace table_group in sql file: update_ddl_2.4.4.sql
sed 's/$VAR_TABLEGROUP_NAME/meta_database/g' ../../sqls/update_ddl_2.4.4.sql > ../../sqls/real/update_ddl_2.4.4.sql
executing real sql script: real/update_ddl_2.4.4.sql
replace table_group in sql file: update_ddl_2.4.5.sql
sed 's/$VAR_TABLEGROUP_NAME/meta_database/g' ../../sqls/update_ddl_2.4.5.sql > ../../sqls/real/update_ddl_2.4.5.sql
executing real sql script: real/update_ddl_2.4.5.sql
replace table_group in sql file: update_ddl_2.5.0.sql
sed 's/$VAR_TABLEGROUP_NAME/meta_database/g' ../../sqls/update_ddl_2.5.0.sql > ../../sqls/real/update_ddl_2.5.0.sql
executing real sql script: real/update_ddl_2.5.0.sql
replace table_group in sql file: update_ddl_2.5.1.sql
sed 's/$VAR_TABLEGROUP_NAME/meta_database/g' ../../sqls/update_ddl_2.5.1.sql > ../../sqls/real/update_ddl_2.5.1.sql
executing real sql script: real/update_ddl_2.5.1.sql
replace table_group in sql file: update_ddl_2.5.4.sql
sed 's/$VAR_TABLEGROUP_NAME/meta_database/g' ../../sqls/update_ddl_2.5.4.sql > ../../sqls/real/update_ddl_2.5.4.sql
executing real sql script: real/update_ddl_2.5.4.sql
replace table_group in sql file: update_ddl_3.1.0.sql
sed 's/$VAR_TABLEGROUP_NAME/meta_database/g' ../../sqls/update_ddl_3.1.0.sql > ../../sqls/real/update_ddl_3.1.0.sql
executing real sql script: real/update_ddl_3.1.0.sql
replace table_group in sql file: update_ddl_3.1.1.sql
sed 's/$VAR_TABLEGROUP_NAME/meta_database/g' ../../sqls/update_ddl_3.1.1.sql > ../../sqls/real/update_ddl_3.1.1.sql
executing real sql script: real/update_ddl_3.1.1.sql
replace table_group in sql file: update_ddl_3.1.2.sql
sed 's/$VAR_TABLEGROUP_NAME/meta_database/g' ../../sqls/update_ddl_3.1.2.sql > ../../sqls/real/update_ddl_3.1.2.sql
executing real sql script: real/update_ddl_3.1.2.sql
replace table_group in sql file: update_ddl_3.2.0.sql
sed 's/$VAR_TABLEGROUP_NAME/meta_database/g' ../../sqls/update_ddl_3.2.0.sql > ../../sqls/real/update_ddl_3.2.0.sql
executing real sql script: real/update_ddl_3.2.0.sql
run sql ALTER TABLE compute_vpc AUTO_INCREMENT = 1001; failed, got exception Traceback (most recent call last):
  File "/home/admin/ocp-init/src/ocp-init/common.py", line 201, in source_sql_file
    run_sql_no_result(sql, config)
  File "/home/admin/ocp-init/src/ocp-init/common.py", line 73, in run_sql_no_result
    run_sql(sql, db_config, True)
  File "/home/admin/ocp-init/src/ocp-init/common.py", line 57, in run_sql
    cursor.execute(sql)
  File "/usr/lib64/python2.7/site-packages/mysql/connector/cursor.py", line 569, in execute
    self._handle_result(self._connection.cmd_query(stmt))
  File "/usr/lib64/python2.7/site-packages/mysql/connector/connection.py", line 590, in cmd_query
    result = self._handle_result(self._send_cmd(ServerCmd.QUERY, query))
  File "/usr/lib64/python2.7/site-packages/mysql/connector/connection.py", line 478, in _handle_result
    raise errors.get_exception(packet)
ProgrammingError: 1146 (42S02): Table 'meta_database.compute_vpc' doesn't exist

 create_metadb.py:21: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(file)
/home/admin/ocp-init/src/ocp-init/generate/gen_dynamic_config_properties.py:14: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(file)
/home/admin/ocp-init/src/ocp-init/generate/yml_to_table.py:69: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(file)
No handlers could be found for logger "root"
Traceback (most recent call last):
  File "create_metadb.py", line 183, in <module>
    init_ocp(filename)
  File "create_metadb.py", line 64, in init_ocp
    init_meta_db(meta_db_connect_info)
  File "create_metadb.py", line 87, in init_meta_db
    metadb.source_sqls(db_connect_info, sqls)
  File "/home/admin/ocp-init/src/ocp-init/metadb.py", line 436, in source_sqls
    common.source_sql_file(config, "../../sqls/{}".format(real_file), force=force)
  File "/home/admin/ocp-init/src/ocp-init/common.py", line 211, in source_sql_file
    raise e
mysql.connector.errors.ProgrammingError: 1146 (42S02): Table 'meta_database.compute_vpc' doesn't exist

```

**Comment by askdb** [comment]
感谢反馈，是ocp3.3.0里一个已知的问题，咨询研发老师，在ocp3.3.1里会修复。
目前可以先去ocp meta 租户的库中确认一下是否已经存在compute_vpc 这张表，如果存在，把其他的表删除，仅保留compute_vpc，然后重新初始化ocp。

**Comment by chris-sun-star** [comment]
fixed in ocp-3.3.0-ce-bp1
